docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Development with local Grafana"
++++
+
+# Development with local Grafana
+
+This guide allows you to setup a development environment where you run Grafana and your plugin locally. With this, you will be able to see your changes as you add them.
+
+## Run Grafana in your host
+
+If you have git, Go and the required version of NodeJS in your system, you can clone and run Grafana locally:
```

Comment:
Maybe refer to https://github.com/grafana/grafana/blob/HEAD/contribute/developer-guide.md instead for the first steps.

Reply:
Agreed.

Reply:
:+1: